src/ucp/api/ucp_def.h

```diff
@@ -234,5 +234,15 @@ typedef void (*ucp_send_callback_t)(void *request, ucs_status_t status);
 typedef void (*ucp_tag_recv_callback_t)(void *request, ucs_status_t status,
                                         ucp_tag_recv_info_t *info);
 
+/**
+ * @ingroup UCP_WORKER
+ * @brief UCP worker wakeup events mask.
+ *
+ * The enumeration allows specifying which events are expected on wakeup
+ */
+typedef enum ucp_wakeup_event_types {
+    UCP_WAKEUP_TX_COMPLETION   = UCS_BIT(0), /**< TX-completions */
+    UCP_WAKEUP_RX_AM           = UCS_BIT(1)  /**< RX-completions */
```

Comment:
BTW, looks like here possible an issue with rndv-read protocol - let's say, receiver ask only RECV events, so event will be generated only for RTS packet - this is a false event in wrong time